azure-functions/src/commands/run.rs

```diff
@@ -75,63 +128,57 @@ impl<'a> Run<'a> {
     pub fn execute(&self, mut registry: Registry<'static>) -> Result<(), String> {
         ctrlc::set_handler(|| {}).expect("failed setting SIGINT handler");
 
-        println!(
-            "Connecting to Azure Functions Host at {}:{}",
-            self.host, self.port
-        );
+        let uri: Uri = format!("http://{0}:{1}", self.host, self.port)
```

Comment:
Is there a more descriptive name than `uri`? Or is the generality good?

Reply:
Maybe `azure_functions_host`?

Reply:
I think perhaps `host_uri` is best.  I'll make that change.